engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnBehaviorsImpl.java

```diff
@@ -54,14 +54,17 @@ public BpmnBehaviorsImpl(
     final StateWriter stateWriter = writers.state();
     final var commandWriter = writers.command();
     this.expressionBehavior = expressionBehavior;
+    final ProcessEngineMetrics metrics = new ProcessEngineMetrics(zeebeState.getPartitionId());
```

Comment:
🔧 The type is already clear from calling the constructor. You can let Java infer the type.

```suggestion
    final var metrics = new ProcessEngineMetrics(zeebeState.getPartitionId());
```